### PR TITLE
[website_docs] fix broken links, copyedits and file renames

### LINKED ISSUE #1104: website_docs contain invalid links
When I try to build the OTel website docs using the Ruby `website_docs` @ HEAD, I get these errors (even after #1081 is included):

```console
$ cd opentelemetry.io
$ npm run update:submodule content-modules/opentelemetry-ruby  # Get Ruby docs @ HEAD
...
$ GET="no" npm run check-links
...tmp/bin/htmltest --skip-external
Skipping the checking of external links.
htmltest started at 10:33:57 on public
========================================================================
docs/instrumentation/ruby/configuring_automatic_instrumentation/index.html
  target does not exist --- docs/instrumentation/ruby/configuring_automatic_instrumentation/index.html --> manual_instrumentation
docs/instrumentation/ruby/getting_started/index.html
  target does not exist --- docs/instrumentation/ruby/getting_started/index.html --> auto-instrumentation
  target does not exist --- docs/instrumentation/ruby/getting_started/index.html --> configure-specific-libraries
  target does not exist --- docs/instrumentation/ruby/getting_started/index.html --> configuring_automatic_instrumentation#configuring-specific-instrumentation-libraries
========================================================================
✘✘✘ failed in 466.748473ms
4 errors in 538 documents
...
```

/cc @cartermp @austinlparker

### PR DESCRIPTION
**Pull request by chalin**
- Closes #1104
- Contributes to:
  - https://github.com/open-telemetry/opentelemetry.io/issues/997
  - https://github.com/open-telemetry/opentelemetry.io/issues/780
- Fixed a typo and copyedited the last section of Getting started.
- Related: https://github.com/open-telemetry/opentelemetry.io/pull/1029

With these changes, the OTel website build is successful (i.e., #1104 is fixed):

```console
$ GET="no" npm run check-links
...
Skipping the checking of external links.
htmltest started at 05:02:44 on public
========================================================================
✔✔✔ passed in 507.334376ms
tested 540 documents
```

/assign @cartermp 
/cc @austinlparker

### CODE REVIEW

File: website_docs/getting-started.md
Comment (chalin) [inline website_docs/getting-started.md:88]: @cartermp - this entire page talks about automatic instrumentation and that same link is given elsewhere in the page, so it seemed odd to me to mention it again here in the list of "more things to look into". Thoughts?

File: website_docs/getting-started.md
Comment (chalin) [inline website_docs/getting-started.md:86]: @cartermp - note that I converted these paragraphs into a list.

File: website_docs/getting-started.md
Comment (cartermp) [reply website_docs/getting-started.md:88]: Makes sense to me.

**Review by cartermp — APPROVED** [review]

**Comment by cartermp** [comment]
(fyi maintainers - I hit approve since I think @chalin's changes make sense w.r.t. what we're doing with docs all-around. I recognize that this requires _your_ approval to move forward 🙂)

**Review by robertlaurin — APPROVED** [review]

**Pull request merged**
